src/Data/Cursor/PostObjectCursor.php

```diff
@@ -230,6 +230,17 @@ private function compare_with_meta_field( string $meta_key, string $order ) {
 				$key = "mt{$index}.meta_value";
 			}
 		}
+
+		/**
+		 * Allow filtering the meta key used for cursor based pagination
+		 *
+		 * @param string $key       The meta key to use for cursor based pagination
+		 * @param string $meta_key  The original meta key
+		 * @param string $meta_type The meta type
+		 * @param string $order     The order direction
+		 * @param object $cursor    The PostObjectCursor instance
+		 */
+		$key = apply_filters( 'graphql_post_object_cursor_meta_key', $key, $meta_key, $meta_type, $order, $this );
```

Comment:
@kidunot89 can you use this existing filter? https://github.com/wp-graphql/wp-graphql/blob/develop/src/Data/Cursor/CursorBuilder.php#L58-L68

I _believe_ you should be able to accomplish the same thing and we wouldn't have to add new filters for each Cursor class.

Reply:
+1 for keeping this generic.

Not sure the original `meta_key` has a strong use case this late in the lifecycle, or at least not one that that justifies a bad signature (appending the param after the instance) or a breaking change.

If there is something I'm missing then a _long term_ goal would be to place the filter in the `AbstractCursor`, so any custom or extended cursors would maintain interoperability without tech debt, but I don't see an easy nonbreaking path so I'm down with duplication as a short term solve and adding a refactor to my evergrowing 2.0 wishlist